TaskManager/Home/View/TaskTableViewCell.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TaskTableViewCell.swift
+//  TaskManager
+//
+//  Created by user on 14.09.2022.
+//
+
+import UIKit
+
+class TaskTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var circleLabel: UILabel!
+    @IBOutlet weak var taskLabel: UILabel!
+    
+    static let identifire = String(describing: TaskTableViewCell.self)
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        setupCircleLabel()
+        setupTaskLabel()
+    }
+    
+    private func setupCircleLabel() {
+        circleLabel.layer.cornerRadius = circleLabel.frame.width / 2
+        circleLabel.layer.masksToBounds = true
+        let color = UIColor.init(netHex: 0xD44F4F)
```

Comment:
lets create static names for the colors you are using in the project. Add to UIColor extension